docs/docs/integrations/third_party_extensions.md

```diff
@@ -0,0 +1,17 @@
+---
+id: third_party_extensions
+title: Third Party Extensions
+---
+
+Beeing open source, others have the possibility to modify and extend the rich functionality Frigate already offers.
+This page is meant to be a overview over additions one can make to the home NVR setup. The list is not exhaustive and can be extended via PR to the Frigate docs.
+
+:::caution
+This page does not recommend or rate the presented projects.
+Please use your own knowledge to assess and vet them before you install anything on your system.
+
+:::
+
+## [Double Take](https://github.com/jakowenko/double-take)
```

Comment:
Suggest move the URL from the heading to the paragraph text to be visually consistent with the rest of the documentation (and more obvious)